### PR TITLE
JAMES-2914 Add serialization integration tests for webadmin

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultDeleteTask.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 public class DeletedMessagesVaultDeleteTask implements Task {
 
-    static final TaskType TYPE = TaskType.of("deletedMessages/delete");
+    public static final TaskType TYPE = TaskType.of("deletedMessages/delete");
```
Comment (mbaechler) [inline server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultDeleteTask.java]: by using this constant in tests, we make it easy to change the value. Is it a property we want? I would rather say we want to make it hard to break the existing endpoint path

Comment (rouazana) [reply server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultDeleteTask.java]: I hesitated, but you are right, I will make the change

Comment (rouazana) [reply server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultDeleteTask.java]: fixed

**Comment by rouazana** [comment]
merged

**Pull request closed without merging**